tests/launcher_test.py

```diff
@@ -235,7 +227,7 @@
 
     testSuccessful=True
 finally:
-    TestHelper.shutdown(cluster, walletMgr, testSuccessful, killEosInstances, killWallet, keepLogs, killAll, dumpErrorDetails)
+    TestHelper.shutdown(cluster, walletMgr, testSuccessful, dumpErrorDetails)
 
 exitCode = 0 if testSuccessful else 1
 exit(exitCode)
```

Comment:
Should this file end with newline while it is being modified?

Reply:
Yes.  Added.

Reply:
74b9300